generators/app/index.js

```diff
@@ -147,8 +147,12 @@ function toCamelCase(aString) {
   var tokens = aString.split(" ");
 
   var camelString = "";
-  tokens.map(function(token) {
-    camelString += token[0].toUpperCase();
+  tokens.map(function(token, index) {
+    if (index) {
+      camelString += token[0]; // Keep the first letter of the first word as it is
```

Comment:
I think you should not leave this first letter as it is. Its format should be consistent. Either to lower case (for camelCase) or to upper case (for PascalCase, a subset of camelCase).
Also, I don't see any problem with the Pascal Case as it was before. Since it's getting appended after *BiojsComponent*, PascalCase seems good for it (lowerCased first letter will create confusion).

Reply:
@awasthishubh Thanks for the review. I changed it to camelCase as I have also removed BiojsComponent which was appended before it. Now if you enter a name like - "Genome Browser". The tool name will be "GenomeBrowser" and not "BiojsComponentGenomeBrowser". Anyway, I will discuss once again and make changes accordingly.